### DISCUSSION TITLE
Invitation to the 1st OpenRadioss Users Meeting in Paris on 22 Nov 2022 !

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#787] [Announcements]
Dear all,
I'm pleased to invite you to our 1st OpenRadioss Users Meeting in Paris on the 22nd of November !
It will be hold as a hybrid in-person remote OpenRadioss workshop event, starting at 2pm (Paris time).

If by chance you are around Paris, please, see below the registration for in-person in Paris and the detailed agenda of the day: Altair Meet’Up + OpenRadioss workshop: https://web.altair.com/fr/altair-meetup-lyon-toulouse-paris

Otherwise, the **link for the online workshop**: 
https://attendee.gotowebinar.com/register/5546513221289609996

After registering, you will receive a confirmation email containing information about joining the webinar.

Interested ? Here is the agenda:
14:00 - 14:45 | OpenRadioss 1: Biomechanics & Safety
14:45 - 15:30 | OpenRadioss 2: Material & Failure - Battery
15:30 - 16:15 | OpenRadioss 3: Fluid Structure Interaction
16:15 - 17:00 | OpenRadioss 4: HPC & Performance
Workshop language: ENGLISH

Looking forward to talk with you :-)

Thanks and best regards,
Marian

**Comment by MarkusSchulten** [comment]
Dear Mr. Bulla,
will there be a recording of the sessions, if someone is not able to participate at the online workshop at the specific time?
Kindly regards
Markus Schulten

**Reply by MarianBulla** [reply]
Dear Mr. Schulten,  
Yes, we will do some recordings. But I'm not sure, if we will be able to publish all of them. 
It depends on the owners of the talks.
We need the signed agreements (their "ok") for further publishing.
Maybe not all talks will be published.

Anyway, it is still in preparation. We are working on that.

I will let you know, after the meeting, where it will be published :-)

Thanks and best regards,
Marian

**Reply by MarkusSchulten** [reply]
Dear Mr. Bulla,
thank you very much for the detailed information. So I will do the registration for your workshop.
Kindly regards
Markus Schulten

**Reply by MarianBulla** [reply]
Dear Mr. Schulten,
you're welcome :-)
Please use the link above mentioned at "...link for the online workshop...".

Looking forward to meet you virtually at the 1st OpenRadioss users meeting in Paris, tomorrow.

Thanks and best regards,
Marian
